### PR TITLE
re-enable ARM runs

### LINKED ISSUE #333: ARM64 CI Leg fails due to missing python3
So far all the ARM64 machines had python3 installed, now they don't.

Related issues:

https://github.com/dotnet/core-eng/issues/5338
https://github.com/dotnet/core-eng/issues/5220

### PR DESCRIPTION
**Pull request by adamsitnik**
fixes #333

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -126,21 +126,21 @@ jobs:
...
+    parameters:
+      osName: ubuntu
+      osVersion: 1604
+      kind: micro
+      architecture: arm64
+      pool: Hosted Ubuntu 1604
+      queue: Ubuntu.1604.Arm64.Open
+      container: ubuntu_1604_arm64_cross_container
+      csproj: src/benchmarks/micro/MicroBenchmarks.csproj
+      categories: ['coreclr', 'corefx']
```
Comment (jorive) [inline azure-pipelines.yml]: Don't we want to run `mldotnet` on ARM64 too?

Comment (adamsitnik) [reply azure-pipelines.yml]: ML.NET has a lot of native dependencies, I don't know if ARM is supported

Comment (jorive) [reply azure-pipelines.yml]: We should find out, but this can be done separately.

Comment (adamsitnik) [reply azure-pipelines.yml]: It's not supported yet https://github.com/dotnet/machinelearning/issues/1662#issuecomment-440378848

**Comment by adamsitnik** [comment]
> It seems like Ubuntu18.04 queue is dead, so all Linux jobs are timing out in CI

it explains why the Ubuntu jobs have failed ;)

**Comment by adamsitnik** [comment]
Clicking re-run failed tests in AzDO does not work, so I need to close and re-open it...

**Comment by billwert** [comment]
Closing this as we have a different queue / set of ARM64 hardware that's coming online.

**Pull request closed without merging**